pom.xml

```diff
@@ -216,6 +225,12 @@
 		</dependency>
 
 
+		<dependency>
+			<groupId>org.javassist</groupId>
+			<artifactId>javassist</artifactId>
+			<version>3.21.0-GA</version>
+		</dependency>
+
```

Comment:
what do we need this for ? (do we need it in the future ? -- I see it seems to be excluded above)

Reply:
It's related with javaassist's problem with java 8. The problem is discussed in [this SO post](https://stackoverflow.com/questions/24281235/error-creating-entitymanagerfactory-due-to-error-tying-to-scan-jar-file).

Reply:
ok -- why do we need javassist ? For hibernate ?

Reply:
Yes, without javassist hibernate throws an exception in our integration tests:

```
Tests run: 3, Failures: 0, Errors: 1, Skipped: 2, Time elapsed: 1.151 sec <<< FAILURE!
test(rcms.utilities.daqexpert.processing.JobManagerIT)  Time elapsed: 0.035 sec  <<< ERROR!
java.lang.NoClassDefFoundError: javassist/bytecode/ClassFile
        at org.hibernate.ejb.packaging.AbstractJarVisitor.checkAnnotationMatching(AbstractJarVisitor.java:243)
        at org.hibernate.ejb.packaging.AbstractJarVisitor.executeJavaElementFilter(AbstractJarVisitor.java:209)
        at org.hibernate.ejb.packaging.AbstractJarVisitor.addElement(AbstractJarVisitor.java:170)
```

Reply:
ok, thanks for the clarification !